docs/Gopkg.toml.md

```diff
@@ -192,10 +192,12 @@ The following are the current available options:
 
 Out of an abundance of caution, dep non-optionally preserves files that may have legal significance.
 
-Pruning is disabled by default. It can be enabled by setting them to `true` at the root level.
+Pruning options are disabled by default. However, generating a `Gopkg.toml` via `dep init` will enable `go-tests` and `unused-packages` prune options by default.
```

Comment:
I think

> `dep init` will **add lines to** enable `go-tests` and `unused-packages` prune options by default.

Is more clear, just so its clear its not magical.

Reply:
Concur

Reply:
What about:
> However, generating a `Gopkg.toml` via `dep init` will **add lines to** enable `go-tests` and `unused-packages` ~prune options by default~.

Unless the trailing 'by default' is referring to how projects inherit from the root settings - In that case, I think 'at the root level' is more clear.

Reply:
I think we should keep the `prune options` part and only remove the `by default` part.